Python3.5:-
Asyncio in python:
    async def fun():
        print("Started fun coroutine")
        print("Ebd of fun corotuine")

    =>  Above piece code creates coroutine.
        Either ->
            foo()   => this call creates a coroutine          #  RuntimeWarning: coroutine 'foo' was never awaited
        or ->
            await foo()
        
    =>  coroutine cannot run itself, we need to drive it from another function as run(below function) or use await foo().
        def run(coro):
            try:
                coro.send(None)
            except StopIteration as evalue:
                return evalue
            except Exception:
                raise
        run(fun())  => will work from here.
    
        
